packages/x-date-pickers-pro/src/internal/hooks/useStaticRangePicker/useStaticRangePicker.tsx

```diff
@@ -21,7 +21,7 @@ import { DateRange } from '../../models/range';
 import { useRangePosition } from '../useRangePosition';
 
 const PickerStaticLayout = styled(PickersLayout)(({ theme }) => ({
-  overflow: 'hidden',
+  overflow: 'auto',
```

Comment:
Without https://next.mui.com/x/react-date-pickers/date-range-picker/#add-shortcuts

Reply:
Could we maybe extract this and the change in `useStaticPicker` into a separate PR?
It does indeed impact the end result of the docs, but in essence, it mainly targets the components and needs special and different attention as well as a separate PR title to bring more clarity via changelog entry. 🤔

Reply:
OK